src/fides/api/ops/models/policy.py

```diff
@@ -37,8 +37,7 @@ class CurrentStep(EnumType):
     access = "access"
     erasure = "erasure"
     consent = "consent"
-    erasure_email_post_send = "erasure_email_post_send"
-    consent_email_post_send = "consent_email_post_send"
+    email_post_send = "email_post_send"
```

Comment:
Just thinking about implications of what happens if we had a failed privacy request in the "consent_email_post_send" state. If we went to resume that email from failure, I think it would fail because we couldn't serialize that from what is stored in Redis, and then we couldn't resume it either.  

But I think this is an edge case, as no one was using the erasure_email_post_send, and the consent_email_post_send is new. Just a callout to be careful when we're updating existing Pydantic fields that are related to serializing data that may be already persisted.

Reply:
Thanks for the callout, I'll keep that in mind for the future.

Reply:
It's also a reminder to myself haha I don't think I've been paying attention here